apps/web/src/components/layout/left-sidebar/DriveEnvAppPane.tsx

```
@@ -0,0 +1,423 @@
+              envId={envId}
+            />
+          )}
+        </div>
+      )}
+
+      <AlertDialog open={confirmingUnpublish} onOpenChange={setConfirmingUnpublish}>
+        <AlertDialogContent>
+          <AlertDialogHeader>
+            <AlertDialogTitle>Unpublish “{envName}”?</AlertDialogTitle>
+            <AlertDialogDescription>
+              The live URL stops resolving and the app's hosting is torn down, including any
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)